src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,42 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int CAPACITY = 10;
+    private int size = 0;
```

Comment:
we may not initialise it explicitly - the default value of int is 0
```suggestion
    private int size;
```